<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>273</width>
      <height>545</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>273</width>
      <height>545</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>273</width>
      <height>545</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border: dark; border-radius:19px</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="Screen" native="true">
       <property name="minimumSize">
        <size>
         <width>249</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>249</width>
         <height>220</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>1000</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: black</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="batteryLabel">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;</string>
          </property>
          <property name="text">
           <string>                              Battery:100%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Display" name="stackedWidget">
          <property name="minimumSize">
           <size>
            <width>228</width>
            <height>164</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>228</width>
            <height>164</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;border-radius: 0px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="currentIndex">
           <number>8</number>
          </property>
          <widget class="QWidget" name="PowerOffPage">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>164</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white;</string>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>211</width>
              <height>121</height>
             </rect>
            </property>
            <property name="text">
             <string>Please Turn The Power On</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="MainMenuPage">
           <property name="styleSheet">
            <string notr="true">background-color:white;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">background:rgb(73, 144, 255);</string>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;MAINMENU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="MainMnueWidget">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="ProgramPage">
           <property name="styleSheet">
            <string notr="true">background-color:white;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">background:rgb(73, 144, 255);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;PROGRAM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="programWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="FrequencyPage">
           <property name="styleSheet">
            <string notr="true">background-color:white;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="styleSheet">
               <string notr="true">background:rgb(73, 144, 255);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;FREQUENCY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="frequencyWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CountDownPage">
           <property name="styleSheet">
            <string notr="true">background-color:white;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="countDownTitle">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="countDownLabel">
              <property name="styleSheet">
               <string notr="true">font: 48pt;qproperty-alignment: AlignCenter;text-align: center;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="picturePage">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="picturPageTitle">
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Plase place the device on skin </string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="PowerPage">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Power&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="powerPageLabel">
              <property name="styleSheet">
               <string notr="true">font: 48pt;qproperty-alignment: AlignCenter;text-align: center;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="powerPageBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CountUpPage">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>164</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>164</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="countUpTitle">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>191</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="countUpLabel">
              <property name="styleSheet">
               <string notr="true">font: 48pt;qproperty-alignment: AlignCenter;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="HistoryPage">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="HistoryPageTitle">
              <property name="styleSheet">
               <string notr="true">background:rgb(73, 144, 255);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;History&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="historyWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="HistoryListPage">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListWidget" name="historyListWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Control" native="true">
       <property name="minimumSize">
        <size>
         <width>211</width>
         <height>291</height>
        </size>
       </property>
       <widget class="Buttons" name="return_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
       <widget class="Buttons" name="menu">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
       <widget class="Buttons" name="off">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>off</string>
        </property>
       </widget>
       <widget class="Buttons" name="down">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>down</string>
        </property>
       </widget>
       <widget class="Buttons" name="left">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>left</string>
        </property>
       </widget>
       <widget class="Buttons" name="right">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>right</string>
        </property>
       </widget>
       <widget class="Buttons" name="up">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>up</string>
        </property>
       </widget>
       <widget class="Buttons" name="ok">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:grey; border-style:inset;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Buttons" name="SkinOnOrOff">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>on/off</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Buttons</class>
   <extends>QPushButton</extends>
   <header location="global">buttons.h</header>
   <slots>
    <signal>signal1()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>Display</class>
   <extends>QStackedWidget</extends>
   <header location="global">display.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
